### ISSUE TITLE
Parameter settings on CIFAR-100

### ISSUE THREAD

**Issue by LiuJia68** [locuslab/fast_adversarial#15]
Hi,

I tried to use this method on CIFAR-100 with the same parameter settings as CIFAR-10. But the results are terrible that the test adversarial accuracies are less than 2%. Do you have any suggestions on how to set up the parameters (epoch, learning rate, and batch size)for CIFAR100? Also, the auxiliary loss is widely used in natural training, do you think it will be helpful if used in fast adversarial training? 

Best wishes,
Jia

**Comment by leslierice1** [comment]
Are you checking for catastrophic overfitting at each epoch during training by running PGD on the first batch of training data? Likely the parameter `alpha` needs to be tuned for the different dataset. I would try fixing all other training hyperparameters and printing the `robust_acc` [here](https://github.com/locuslab/fast_adversarial/blob/54f728755e71857b632882ba6d6cef22f56e2172/CIFAR10/train_fgsm.py#L136), and reducing `alpha` until you see training stabilize.
